wfl/calculators/generic.py

```diff
@@ -52,7 +53,9 @@ def run_op(atoms, calculator, properties=None, output_prefix='_auto_', verbose=F
         at.calc = calculator
         calculation_succeeded = False
         try:
-            at.calc.calculate(at, properties=properties)
+            # explicitly pass system_changes=None because some calculators, e.g. ace.ACECalculator,
```

Comment:
typo

Reply:
fixed in 

> The tests aren't passing - looks like phonopy is missing from `.github/workflows/python-package-conda.yml`?

Yes, I'll fix that.
 
> I've left some small comments, but otherwise looks good. I'll extend `ace_fit.jl` with what I'll use in another pull request.
> 
> What do you think about making a proper (small) julia package instead of ace_fit.jl? It'd make it more modular, testable and easier to develop.

I'm happy either way.  My initial thought was just the simplest thing, analogous to `gap_fit` (although not using the same legacy command line syntax), but if you or someone else likes another approach, that's find by me.